docs/sample_config/default_config.nu

```diff
@@ -256,6 +256,7 @@ let-env config = {
   cd_with_abbreviations: false # set to true to allow you to do things like cd s/o/f and nushell expand it to cd some/other/folder
   case_sensitive_completions: false # set to true to enable case-sensitive completions
   enable_external_completion: true # set to false to prevent nushell looking into $env.PATH to find more suggestions, `false` recommended for WSL users as this look up my be very slow
+  max_external_command_completions: 100
```

Comment:
Maybe we can have a shorter name?  Like `max_external_cmd_completions`

And I think it's better to have a comment for it, like other items.

Reply:
Good call, changed it to `max_external_completions` in line with `enable_external_completion`